persistence/src/test/java/edu/unc/lib/dl/persist/services/edit/EditTitleServiceTest.java

```diff
@@ -162,21 +183,57 @@ public void multipleTitlesInMods() throws Exception {
         document.addContent(new Element("mods", MODS_V3_NS)
                 .addContent(new Element("titleInfo", MODS_V3_NS)
                         .addContent(new Element("title", MODS_V3_NS).setText("original title")))
-                .addContent(new Element("titleInfo", MODS_V3_NS))
-                        .addContent(new Element("title", MODS_V3_NS).setText("a second title")));
+                .addContent(new Element("titleInfo", MODS_V3_NS)
+                        .addContent(new Element("title", MODS_V3_NS).setText("a second title"))));
         when(binaryObj.getBinaryStream()).thenReturn(convertDocumentToStream(document));
 
         service.editTitle(agent, pid, title);
 
         verify(operationsMessageSender).sendUpdateDescriptionOperation(anyString(), pidsCaptor.capture());
         Collection<PID> pids = pidsCaptor.getValue();
-        assertEquals(pids.size(), 1);
+        assertEquals(1, pids.size());
         assertTrue(pids.contains(pid));
+
+        verify(contentObj).setDescription(inputStreamCaptor.capture());
+        Document updatedDoc = getUpdatedDescriptionDocument(inputStreamCaptor.getValue());
+        assertTrue(assertTitleValue(updatedDoc, title));
+        // check that first title is no longer in mods
+        assertFalse(assertTitleValue(updatedDoc, "original title"));
+        // check that second title is unchanged
+        assertTrue(assertTitleValue(updatedDoc, "a second title"));
     }
 
     private InputStream convertDocumentToStream(Document doc) throws IOException {
         ByteArrayOutputStream outStream = new ByteArrayOutputStream();
         new XMLOutputter().output(doc, outStream);
         return new ByteArrayInputStream(outStream.toByteArray());
     }
+
+    private Document getUpdatedDescriptionDocument(InputStream inputStream) {
+        Document document;
+        String modsString;
+
+        try {
+            modsString = IOUtils.toString(inputStream, StandardCharsets.UTF_8);
+        } catch (IOException e) {
+            throw new ServiceException("Unable to covert mods stream to string for " + pid, e);
+        }
+
+        ByteArrayInputStream modsByteArray = new ByteArrayInputStream(modsString.getBytes());
+        SAXBuilder sb = new SAXBuilder(new XMLReaderSAX2Factory(false));
+        try {
+            document = sb.build(modsByteArray);
+        } catch (IOException| JDOMException e) {
+            throw new ServiceException("Unable to build mods document for " + pid, e);
```

Comment:
for test methods/helpers, i generally just let the method `throw Exception` since there isn't going to be any other code relying on the API of the method to be more descriptive.